argilla/src/argilla/webhooks/_event.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+from pydantic import BaseModel
+
+from argilla import Argilla, Dataset, Record, UserResponse
+from argilla._models import DatasetModel, RecordModel, UserResponseModel
+
+
+class WebhookEvent(BaseModel):
+    type: str
+    timestamp: datetime
+    data: dict
+
+    def normalize(self, client: Argilla) -> dict:
```

Comment:
type checking import
```suggestion
    def normalize(self, client: "Argilla") -> dict:
```